CooperationHullMainSite/Views/Shared/_HappeningNext.cshtml

```diff
@@ -1,22 +1,16 @@
-﻿<div class="col-lg-4 col-md-6 col-sm-6 p-5">
-
+﻿<div class="col-xl-4 col-md-6 col-sm-12">
```

Comment:
Still a bit more work to do on this grid. I've added a ticket to the backlog to align it, but it's a bit tough to do right now while sections are still in progress